resources/testdata/glyphs2/BracketTestFontKerning.glyphs

```diff
@@ -0,0 +1,397 @@
+{
+.appVersion = "1342";
+customParameters = (
+{
+name = Axes;
+value = (
+{
+Name = Weight;
+Tag = wght;
+},
+{
+Name = Width;
+Tag = wdth;
+}
+);
+},
+{
+name = "Disable Last Change";
+value = 1;
+}
+);
+familyName = "New Font";
+fontMaster = (
+{
+ascender = 800;
+capHeight = 700;
```

Comment:
Looks possible to minimize the test input, for example do you actually care about these for this test?  What about disable last change, etc?